protocols/kad/src/behaviour.rs

```diff
@@ -286,6 +289,26 @@ where
         }
     }
 
+    /// Gets an iterator over immutable references to all running queries.
```

Comment:
If I am not mistaken `queries.iter` can as well yield queries that are in `Finished` state, right? I guess that could still count as `running`?

Reply:
True, there can certainly be queries which are already "finished" but have not yet been seen as finished by the `Kademlia` behaviour, since that can only be detected on `Kademlia::poll`. I guess we could filter them out here, what do you think?

Reply:
I decided to filter finished queries on this API.